toolbox/py_xnvme_pkg/xnvme/ctypes_mapping.py

```diff
@@ -0,0 +1,4272 @@
+# -*- coding: utf-8 -*-
```

Comment:
I would recommend not having the auto-generated mapping in the repo. As @safl noted offline, the ordering of the output is not stable, so git-diffs are impossible to check. I would recommend just adding it as part of the build process, and publish it as part of a package on PyPi.

Reply:
I agree. They are have been removed now.
The facilitate the build then a ci-job now builds the the sdist package and added as an artifact.
By doing so, then we get to do two things: 1) Automate the build of the package. 2) Test that the runtime-dependencies are met on all the usual targets.